data/autoyast_qam/12-common_base_installation.xml.ep

```diff
@@ -301,8 +304,93 @@
   <partitioning config:type="list">
     <drive>
     <initialize config:type="boolean">true</initialize>
-    <use>all</use>
-    </drive>
```

Comment:
The same comment as above one, can you please run VR for aarch64 platform as well?